integration_tests/tests/forseti/controls/explain.rb

```diff
@@ -60,31 +60,6 @@
             end
         end
 
-            it "should be visible from the command-line" do
-                expect(command("forseti explainer list_members").stdout).to match /serviceaccount\/84605163300-compute@developer.gserviceaccount.com"/
```

Comment:
Why are these list member results removed in this PR?  Do they have anything to do with the adding the buckets?

Reply:
I have removed all the service accounts that do not need to be preserved for the tests to run for clean-up purpose.

Reply:
Then it would be best for this clean-up to be in a separate PR, and not be part of this PR.  So, please remove this change here, and submit separately in a new PR.

Reply:
Some of this service accounts have been deleted as they were in pending deletion phase. So, it needs to be removed in this PR for tests to pass.

Reply:
Can you still break this out into a separate PR, and submit that first to fix the brokenness?  Then, you can submit this later.  The separation helps with later discoverability of the work that's being done.